### PR TITLE
chore: remove unused op_status source and polish error reporting for

### PR DESCRIPTION
**Pull request by kostasrim**
* Remove unused op_status.cc
* Polish error reporting messages in memcached

### CODE REVIEW

**Comment by kostasrim** [comment]
@romange You forgot to remove the empty source file + I split the error reporting for memcached such that we know if it's a generic error or an `unknown command`

File: src/facade/dragonfly_connection.cc [hunk 1]
```diff
@@ -732,7 +734,7 @@ auto Connection::IoLoop(util::FiberSocketBase* peer, SinkReplyBuilder* orig_buil
         DCHECK_GT(io_buf_.AppendLen(), 0U);
       } else if (io_buf_.AppendLen() == 0) {
         // We have a full buffer and we can not progress with parsing.
-        // This means that we have request too large.
+        // This means that we have a too large request.
```
Comment (dranikpg) [inline src/facade/dragonfly_connection.cc:737]: Actually sounds fine to me as `a request too large to be processed` is valid english 🙂

**Comment by romange** [comment]
the current implementation follows the memcached spec and is compatible with Memcached.
This PR breaks the compatibility.

**Comment by romange** [comment]
this is the spec: https://github.com/memcached/memcached/blob/master/doc/protocol.txt

**Comment by kostasrim** [comment]
> this is the spec: https://github.com/memcached/memcached/blob/master/doc/protocol.txt

I did not know this! Thank you for sending it! 

I will close the PR since `op_status.cc` now contains staff because of https://github.com/dragonflydb/dragonfly/pull/1741

**Pull request closed without merging**